doc/_templates/layout.html

```diff
@@ -0,0 +1,32 @@
+{% extends "!layout.html" %}
+{% block sidebartitle %}
+
+  {% if logo and theme_logo_only %}
+    <a href="{{ pathto(master_doc) }}">
+  {% else %}
+    <a href="{{ pathto(master_doc) }}" class="icon icon-home" alt="{{ _("Documentation Home") }}"> {{ project }}
+  {% endif %}
+
+  {# this div is instead of the usual logo. The height is picked to avoid a jump
+     when mathjax renders #}
+  <div class="math notranslate nohighlight" style="height: 4.4em; display: table">
```

Comment:
```suggestion
  <div class="math notranslate nohighlight" style="height: 4.4em; display: table; color: rgba(0, 0, 0, 0)">
```

Reply:
This could avoid the flashing by hiding the source code and it has no effect on the white MathJax.

Reply:
Clever. I assume `color: transparent` works too?

Reply:
```suggestion
  <div class="math notranslate nohighlight" style="height: 4.4em; display: table; color: transparent">
```

Reply:
Need some tests to confirm.

Reply:
I think it's working now?